modules/gapi/samples/face_detection.cpp

```diff
@@ -56,45 +56,45 @@ namespace {
 #define NUM_PTS 5
 
 struct BBox {
-    double x1;
-    double y1;
-    double x2;
-    double y2;
+    int x1;
+    int y1;
+    int x2;
+    int y2;
 
-    cv::Rect getRect() const { return cv::Rect(static_cast<int>(x1),
-                                               static_cast<int>(y1),
-                                               static_cast<int>(x2 - x1),
-                                               static_cast<int>(y2 - y1)); }
+    cv::Rect getRect() const { return cv::Rect((x1),
+                                               (y1),
+                                               (x2 - x1),
+                                               (y2 - y1)); }
 
     BBox getSquare() const {
         BBox bbox;
-        double bboxWidth = x2 - x1;
-        double bboxHeight = y2 - y1;
-        double side = std::max(bboxWidth, bboxHeight);
-        bbox.x1 = static_cast<double>(x1) + (bboxWidth - side) * 0.5;
-        bbox.y1 = static_cast<double>(y1) + (bboxHeight - side) * 0.5;
-        bbox.x2 = bbox.x1 + side;
-        bbox.y2 = bbox.y1 + side;
+        float bboxWidth = static_cast<float>(x2 - x1);
+        float bboxHeight = static_cast<float>(y2 - y1);
+        float side = std::max(bboxWidth, bboxHeight);
+        bbox.x1 = static_cast<int>(static_cast<float>(x1) + (bboxWidth - side) * 0.5f);
```

Comment:
What if `side = bboxHeight` and x1 will be less 0? It is out of image bbox.

Reply:
https://github.com/openvinotoolkit/open_model_zoo/blob/master/demos/face_detection_mtcnn_demo/python/mtcnn_utils.py#L4

Reply:
The same as the original demo